### PR TITLE
Add native-base / antd-mobile for native UI

### PR DESCRIPTION
**Pull request by mitjade**
- refactored post list to use FlatList
- implemented infinite scroll with onEndReached
- added swipe action for deleting post
- refactored edit post to use native-base / antd-mobile components

### CODE REVIEW

File: src/client/modules/common/components/native/ui-native-base/components/FormButton.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, Text } from 'react-native';
+import { Button } from 'native-base';
+
+const FormButton = ({ children, ...props }) => {
```
Comment (mairh) [inline src/client/modules/common/components/native/ui-native-base/components/FormButton.jsx]: I guess this should be only `Button`. It is not specifically tied to `Form` component.

Comment (mairh) [reply src/client/modules/common/components/native/ui-native-base/components/FormButton.jsx]: ah I see you have another component named `Button`

Comment (mitjade) [reply src/client/modules/common/components/native/ui-native-base/components/FormButton.jsx]: @mairh I needed to style the button a bit differently if in form, so I added another wrapper.

**Comment by mitjade** [comment]
@mairh @vlasenko Are you OK with merging this to master?

**Pull request merged**